tests/integration/go_ethereum/test_goethereum_ipc.py

```diff
@@ -66,10 +66,21 @@ class TestGoEthereumAdminModuleTest(GoEthereumAdminModuleTest):
     def test_admin_peers(w3):
         super().test_admin_peers(w3)
 
-    @pytest.mark.xfail(reason="websockets aren't enabled with our IPC flags")
```

Comment:
Was this `xfail` meant to be removed? 👀

Reply:
The one within the test produces the same effect. I used the reason from other places for consistency, but might merit a deeper look.

Reply:
yep I see that now... as we discussed, let's definitely try to look to see if we can un-`xfail` all these tests at some point as another issue. lgtm for now though 👌